src/main/java/edu/kis/powp/jobs2d/drivers/adapters/DriverToDrawerDottedLineAdapter.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.powp.jobs2d.drivers.adapters;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+/**
+ * driver adapter to drawer with dotted line parameters
+ */
+public class DriverToDrawerDottedLineAdapter implements Job2dDriver {
+	private int startX = 0, startY = 0;
+	private DrawPanelController drawPanelController;
+
+	public DriverToDrawerDottedLineAdapter(DrawPanelController drawPanelController) {
+		super();
+		this.drawPanelController = drawPanelController;
+	}
+
+	@Override
+	public void setPosition(int x, int y) {
+		this.startX = x;
+		this.startY = y;
+	}
+
+	@Override
+	public void operateTo(int x, int y) {
+		ILine line = LineFactory.getDottedLine();
```

Comment:
Unfortunately, this is not a generic solution. (e.g. new line type requires new class, unnecessary direct coupling with LineFactory)